packages/react/transform/swc-plugin-reactlynx-compat/src/lib.rs

```diff
@@ -1 +1,1870 @@
+use convert_case::{Case, Casing};
```

Comment:
Instead of copying the whole file, can we re-use the existing mod or crate? E.g.:

- crates/swc_plugin_* <-- pure Rust crates that contains swc plugins
- packages/react/transform <-- napi-rs binding
- packages/react/transform/swc-plugin-* <-- WASM plugin binding